include/experimental/__p1673_bits/linalg_execpolicy_mapper.hpp

```diff
@@ -12,11 +12,12 @@ inline namespace __p1673_version_0 {
 namespace linalg {
 namespace impl {
 // the execution policy used for default serial inline implementations
-struct inline_exec_t {};
+using inline_exec_t = std::execution::sequenced_policy;
 
 // The execution policy used when no execution policy is provided
 // It must be remapped to some other execution policy, which the default mapper does
-struct default_exec_t {};
+using default_exec_t = std::execution::parallel_policy;
```

Comment:
`// It must be remapped to some other execution policy, which the default mapper does` -- it's important that we not circumvent the default mapper.  Would you consider, thus, reverting this change?

Reply:
If I remember correctly, the execpolicy_mapper did not like this default. I can restore it and show you the error it gave me.